patches/tModLoader/Terraria/ModLoader/ModTile.cs

```diff
@@ -77,6 +79,28 @@ public void AddMapEntry(Color color, LocalizedText name, Func<string, int, int,
 		}
 	}
 
+	/// <summary>
+	/// Manually registers an item to drop for the provided tile styles. Use this for tile styles that don't have an item that places them, but could potentially exist. For example, open door tiles don't have any item that places them, but they can exist and should drop an item corresponding to their style when destroyed.<br/><br/>
```

Comment:
Remove `, but could potentially exist` and `can exist and` and `corresponding to their style`

Instead consider saying `A tile style with no registered drop and no fallback drop will not drop anything when destroyed`